src/pages/LoadingPage.tsx

```diff
@@ -1,25 +1,11 @@
 import { DotLottieReact } from '@lottiefiles/dotlottie-react';
 
-// interface LoadingProps {
-//   //loading: boolean;
-//   //onNext: () => void;
-// }
-
 const LoadingPage: React.FC = () => {
-  //const [loading, setLoading] = useState(true);
-
-  // useEffect(() => {
-	// 	// Simulate a data fetching process
-	// 	setTimeout(() => {
-	// 	//setLoading(false);
-  //   onNext();
-	// 	}, 5000);
-	// }, []);
 
   return (
     <div className='bg-[var(--lightgreen)] flex justify-center items-center w-full h-full'>
         <DotLottieReact
-          src="/src/assets/anime_man.json"
+          src="/src/assets/anima_loading.json"
```

Comment:
I think the issue is with how you import it, you should import from the top as 
```tsx
import loadingAnimation from "../assets/anima_loading.json";
```
And then here you would write:
```tsx
src={loadingAnimation}
```